### ISSUE TITLE
Add Block: Revision History

### ISSUE THREAD

**Issue by bhubbard** [WordPress/gutenberg#40389]
## What problem does this address?
If revisions are enabled, then new blocks should be enabled to display information about revisions. This could be a list of the revisions and who made the changes, or say a total count of revisions.

## What is your proposed solution?
A new block that lists revisions of the current post/page/CPT. I suggest a list view showing the date/time and the person who made the change.

**Comment by carolinan** [comment]
I don't think that this block would be useful for the majority of the WordPress users and I think it should be in a separate plugin.
But this is only my opinion.
